docs/README-osx.md

```diff
@@ -19,3 +19,14 @@ brew install openssl libffi autoconf automake libtool
 ```sh
 brew install leveldb
 ```
+
+4. When you are on `>=OSX 10.15 Catalina` and you encounter error with default `ZSH` shell:
```

Comment:
```suggestion
> If you are on `>=OSX 10.15 Catalina` you may encounter the following error with the default `ZSH` shell. This can be fixed by wrapping the `[dev]` part in quotes.
```